bumble/pandora/security.py

```diff
@@ -294,23 +294,37 @@ async def Secure(
             try:
                 self.log.debug('Pair...')
 
+                wait_for_security = asyncio.get_running_loop().create_future()
+
+                def on_pairing(*_: Any) -> None:
```

Comment:
suggest: you could use the decorator syntax here: 
```
@connection.on("pairing")
def on_pairing(*_: Any) -> None:
  ...
```

Reply:
Done

Reply:
See PR #292 for a proposal of how this could be done.